misc/python/materialize/output_consistency/common/format_constants.py

```diff
@@ -0,0 +1,15 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+CONTENT_SEPARATOR_1 = "==============="
```

Comment:
More descriptive names of the constantsmay help here.

Reply:
✅ Renamed